codex/rest/api.nim

```diff
@@ -209,6 +211,43 @@ proc initDataApi(node: CodexNodeRef, repoStore: RepoStore, router: var RestRoute
   router.api(
     MethodGet,
     "/api/codex/v1/data/{cid}/network") do (
+      cid: Cid, resp: HttpResponseRef) -> RestApiResponse:
```

Comment:
:yellow_circle: OK this is maybe nitpicking but since this generates side-effects at the server (and returns no data), shouldn't this be a POST?

Reply:
A very good point. Fixing now!